xwiki-platform-core/xwiki-platform-index/xwiki-platform-index-ui/src/main/resources/XWiki/DeletedAttachments.xml

```diff
@@ -156,13 +156,13 @@ $response.setContentType('application/json')
   #set($dates = $dateFilter.split('-'))
   #if($dates.size() == 2)
     ## Date range matching
-    #set($sql = "${sql} and datt.date between ? and ? ")
-    #set($discard = $params.add($datetool.toDate($numbertool.toNumber($dates[0]))))
-    #set($discard = $params.add($datetool.toDate($numbertool.toNumber($dates[1]))))
+    #set($sql = "${sql} and datt.date between :start_date and :end_date ")
+    #set($discard = $params.put('start_date', $datetool.toDate($numbertool.toNumber($dates[0]))))
+    #set($discard = $params.put('end_date', $datetool.toDate($numbertool.toNumber($dates[1]))))
```

Comment:
I would use camel case for the parameter names, to be in-line with our code style.